COMETwebapp/Model/Applications.cs

```diff
@@ -103,6 +104,16 @@ private static List<Application> InitializesApplications()
                     ComponentType = typeof(ElementDefinitionTable)
                 },
 
+                new TabbedApplication
+                {
+                    Name = "Multi Model Editor",
```

Comment:
please change name to Model Editor and this one needs to replace the existing one